app/models/service_template_orchestration.rb

```diff
@@ -3,6 +3,24 @@ class ServiceTemplateOrchestration < ServiceTemplate
 
   before_save :remove_invalid_resource
 
+  def self.create_catalog_item(options, _auth_user = nil)
+    transaction do
+      create(options.except(:config_info)).tap do |service_template|
+        config_info = options[:config_info]
+
+        service_template.orchestration_template = if config_info[:template_id]
```

Comment:
@bzwei yeah, good point. if both are required for a valid orchestration template, we should check that both exist, as well as make an update / delete method.